### PR TITLE
Initial Go image

### PR DESCRIPTION
**Pull request by jiexi**
@alexdavid

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM pairforce/base-image
+
+# Taken from Docker's Dockerfile Go repo
+# Install Go
+RUN mkdir -p /goroot
```
Comment (alexdavid) [inline Dockerfile]: I think it might make more sense to install go to /usr/bin/go

**Comment by jiexi** [comment]
@alexdavid comment addressed

**Comment by jiexi** [comment]
@alexdavid tests passing

**Comment by alexdavid** [comment]
Thanks! :ship:

**Pull request closed without merging**